clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -978,7 +1000,7 @@ public boolean hasValidPosition() {
 
             @Override
             public boolean hasPosition() {
```

Comment:
As it produced a bug, could you add some comment for this method?

Reply:
I added some javadoc to the interface